### ISSUE TITLE
Problem with WP 6.1 update and 'Link-Library' plugin

### ISSUE THREAD

**Issue by Uatschitchun** [studio-arrenberg/quartiersplattform#146]
I've got a problem with**:** WP 6.1 **+** Quartiersplattfrom theme **+** [Link Library plugin](https://github.com/ylefebvre/link-library)

My website wasn't available at the root address since automatic update to WP 6.1 (deactivated automatic updates now). Pages like example.com/projekte or example.com/login were working, but not just example.com
![Screenshot_20221104_165743](https://user-images.githubusercontent.com/25088926/200021217-a2c29b28-0e3f-42c8-bfa8-a6b02a1beea3.jpg)

Tried plugin with different themes => all working fine
Tried QP-theme without plugin => all working fine, but no link list (because deactivated)
Tried downgrading WP to 6.0.3 => QP-theme + Link Library working fine again.

Something changed with WP 6.1 rendering a 'Bad Gateway' (502) error.

Enabling wordpress' debug doesn't show anything

**Comment by ylefebvre** [comment]
I have not seen any feedback from you on this issue. Where can I get a copy of this theme to test? If it's a home-grown or paid theme, then the next thing would be to enable the site's WP_DEBUG feature and see what types of errors are generated by your site.

**Comment by Uatschitchun** [comment]
@ylefebvre Sorry, haven't seen your reply. The quartiersplattform theme is free, simply clone

I tried again today, updated to WP 6.1.1 and got the same "page unavailable" error again when enabling "link library" ;(

Together with that I got a whole lotta of these:
`PHP Warning:  Cannot modify header information - headers already sent by (output started at /homepages/34/d77201858/htdocs/wp-hammesberg/wp-content/plugins/advanced-custom-fields-pro/includes/acf-value-functions.php:54) in /homepages/34/d77201858/htdocs/wp-hammesberg/wp-content/themes/quartiersplattform/functions.php on line 3319`

I got it solved with commenting line 3328 in functions.php:
`//add_filter( 'determine_locale', 'qp_language', 10, 1 );`

After that the site is working again with "link library" enabled and the mentioned _Warning_ is gone. Translations do not work anymore after commenting `add_filter`.

There seems something broken with translations here, as https://github.com/studio-arrenberg/quartiersplattform/issues/147 & https://github.com/studio-arrenberg/quartiersplattform/issues/148 show.

So, @ylefebvre, it seems not to be a problem with your plugin, alas it is triggering it.

**Comment by ylefebvre** [comment]
Very odd. The only time that Link Library does anything related to locale is by using @setlocale but this only happens when you're saving your library settings. So I really don't understand why this is causing a problem. Did you have any luck contacting the author of the quartiersplattform theme?

**Comment by Uatschitchun** [comment]
> Did you have any luck contacting the author of the quartiersplattform theme?

Yes, this is the github belonging to _Quartiersplattform_ theme :-)

**Comment by ylefebvre** [comment]
My bad. I did not see that. But I also don't see any resolution here since you posted back in March. Is the conflict still present or the issue was just not closed?
